core/app_test.go

```diff
@@ -229,16 +232,16 @@ func argTestCleanup(oldArgs []string) {
 
 func testParseExpectError(t *testing.T, testJSON string, expected string) {
 	os.Args = []string{"this", "-config", testJSON, "/testdata/test.sh", "test", "--debug"}
-	if _, err := LoadConfig(); err != nil && !strings.Contains(err.Error(), expected) {
+	if _, err := LoadApp(); err != nil && !strings.Contains(err.Error(), expected) {
 		t.Errorf("Expected %s but got %s", expected, err)
 	}
 }
 
 func validateParseError(t *testing.T, input []byte, matchStrings []string) {
-	if cfg, err := unmarshalConfig([]byte(input)); err != nil {
+	if cfg, err := config.UnmarshalConfig([]byte(input)); err != nil {
```

Comment:
That we had to export this method from the `config` package makes me question whether this test is at the right level of abstraction.